### PR TITLE
First draft of device failure prediction WG charter

### PR DESCRIPTION
**Pull request by dmick**
Signed-off-by: Dan Mick <dmick@redhat.com>

### CODE REVIEW

**Comment by erikerlandson** [comment]
cc @MichaelClifford @Shreyanand

**Comment by schwesig** [comment]
I would give a lgtm, but does this need some more review?
@erikerlandson

**Review by schwesig — COMMENTED** [review]
looking good to me

**Comment by erikerlandson** [comment]
@dmick  can you change name of the directory to `wg-device-failure-prediction`?  The WG directories all start with `wg-`

**Comment by erikerlandson** [comment]
@dmick   also we need an entry under the workinggroups section here:
https://github.com/open-services-group/community/blob/main/sigs.yaml#L143

**Comment by erikerlandson** [comment]
@dmick also it looks like I was wrong, we just need to list the orgs in "stakeholders"

File: device-failure-prediction/charter.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+* A plan for future work in modeling device behavior
+* At least one working failure prediction model
+* Recommendations on usage of T13 [SMART] and Seagate [FARM] data in modeling, and potential future modifications to drive vendor telemetry standards
+* proposal for a productization roadmap (if predictive modeling turns out to be sufficiently useful)
+
+[SMART]:http://t1.daumcdn.net/brunch/service/user/axm/file/zRYOdwPu3OMoKYmBOby1fEEQEbU.pdf
+[FARM]:https://github.com/Seagate/openSeaChest_LogParser/blob/develop/docs/FARM_Specification.pdf
+
+## Stakeholders
```
Comment (chauhankaranraj) [inline device-failure-prediction/charter.md:42]: Quick question, so I had initially worked on creating the failure prediction models that are in Ceph upstream rn, and have been working with the Ceph team on some other initiatives too (Seagate collab, telemetry data, [disk-health-predictor](https://github.com/aicoe-aiops/disk-health-predictor) standalone module etc). So should I add myself as a stakeholder here? Or would it be better to add myself as a member?

/cc @MichaelClifford @erikerlandson

Comment (erikerlandson) [reply device-failure-prediction/charter.md:42]: @chauhankaranraj  the stakeholders are org-level, we should assemble a list of individuals as members

Comment (chauhankaranraj) [reply device-failure-prediction/charter.md:42]: Gotcha, in that case I'll add myself as a non-stakeholder member later :)

**Review by schwesig — APPROVED** [review]
/lgtm
/approve

**Comment by dmick** [comment]
Updated with Erik's comments, and added github configuration

**Comment by erikerlandson** [comment]
@Shreyanand   can you add @dmick as a org member, to satisfy peribolos?

**Comment by dmick** [comment]
ping @Shreyanand

**Comment by Shreyanand** [comment]
@harshad16 @tumido how can we fix the peribolos test here?

**Comment by harshad16** [comment]
as there is a new team creation, the peribolos should be overridden, as it cant do a dry run of the creation of the team.
the owner has to done override by using following command:

/override peribolos

**Comment by dmick** [comment]
The bots have been venerated properly!  Hurrah!

**Pull request merged**